stdlib/array.mli

```diff
@@ -256,6 +256,20 @@ val fast_sort : ('a -> 'a -> int) -> 'a array -> unit
 *)
 
 
+(** {6 Iterators} *)
+
+val to_seq : 'a array -> 'a Seq.t
+(** Iterate on the array, in increasing order
```

Comment:
You should specify that mutation during iteration is not specified (as you did for Buffer).